operation/tube_handler.go

```diff
@@ -22,7 +22,7 @@ func NewTubeHandler(
 		for {
 			select {
 			case <-ticker.C:
-				log.Debug("House Keeping Started for: ", name)
+				// log.Debug("House Keeping Started for: ", name)
```

Comment:
This is not required the problem here is that the logger used is not the correct one because the global variable is getting replaced by Gos` default logger. If it was the correct logger it will not print debug logs by default. Solution is to import the logger explicitly in this file. Please ignore this for now as a permanent solution is coming.